components/automate-ui/src/app/modules/infra-proxy/environment-details/environment-details.component.ts

```diff
@@ -127,7 +129,7 @@ export class EnvironmentDetailsComponent implements OnInit, OnDestroy {
     }
   }
 
-  toDisplay(cookbookVersions: CookbookVersion[]): CookbookVersionDisplay[] {
+  toDisplay(cookbookVersions: Object): CookbookVersionDisplay[] {
```

Comment:
Why this change?

Reply:
while creating the environment or while adding the particular environment we are using cookversion as object type that's why I changed it to object.